tests/qa.py

```diff
@@ -80,6 +80,29 @@ def test_qa_sign_tx_custom_tokens(cmd):
     signatures = cmd.sign_tx(tx)
 
 
+def test_qa_sign_tx_with_token(cmd):
```

Comment:
i can't find where you used this function.

Reply:
The QA file is not meant to run on the automated tests.

The idea of these tests is to run with a dev so the dev can check that all screens (UI/UX) are working as intended.
While we do not have a QA for the Ledger I like to keep this file updated with new methods and features since I also use this `qa.py` for simple tests.